src/mobot/settings.py

```diff
@@ -129,13 +131,16 @@
             'NAME': DATABASE_NAME,
             'USER': DATABASE_USER,
             'PASSWORD': DATABASE_PASSWORD,
-            'HOST': DATABASE_HOST,
             'PORT': DATABASE_PORT,
+            'DISABLE_SERVER_SIDE_CURSORS': True,
```

Comment:
I'm not sure you want to do this.  If you're having problems with concurrency, remember that Django and these kind of older CGI style apps were designed to have a single thread per request and a single database connection per thread.  You may want to looks at other frameworks for the "worker" tasks, or figure out how to spawn multiple threads for concurrent conversations. 

For database efficiency with Django I guess you can set persistent connections:  https://docs.djangoproject.com/en/3.2/ref/databases/#persistent-connections

So something like CONN_MAX_AGE=120

Now this doesn't really solve concurrency, but again we may just be using the wrong tools here.

Reply:
Note, I REALLY DO NOT want to run pgbouncer. A db proxy like this should be completely unnecessary in this day and age.

Reply:
I figured out a way around it :)